### PR TITLE
fix(saas-ui):change in design and structure of application

### PR DESCRIPTION
**Pull request by Deepika516**
## Description
1.) design enhancement 
2.)changes in project structure

Fixes #34 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts [hunk 1]
```diff
@@ -28,6 +28,10 @@ export class OnboardingTenantListComponent
   rowData = [];
   tenants: any;
   leads: any;
+  // for Pagination
+  pagination = true;
+  paginationPageSize = 5;
+  paginationPageSizeSelector = [5, 10, 20, 50, 100];
```
Comment (yeshamavani) [inline projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts:34]: this is not to be done like this 
we need performance improvement.. based on the pagination value only those many rows should be fetched from backend

Comment (yeshamavani) [reply projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts:34]: same everywhere

**Review by yeshamavani — CHANGES_REQUESTED** [review]

File: projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts (hunk 1 above)
Comment (Deepika516) [reply projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts]: removed

File: projects/saas-ui/src/app/on-boarding/on-boarding-service.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ import {
 import {GetPlanAdapter} from './adapters';
 import {Lead, Tenant, Plan} from './models';
 
-import {GetBillingDetails} from '../main/lead-list/commands/get-billing.command';
```
[the remark this answers is not in the document]
Comment (Deepika516) [reply projects/saas-ui/src/app/on-boarding/on-boarding-service.ts:24]: i removed this import actually we are not using this GetBillingDetails command here its by mistake i forgot to remove this used import

File: projects/saas-ui/src/app/main/commands/add-tenant.command.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import {ApiService, IAdapter, PostAPICommand} from '@project-lib/core/api';
 
 import {IAnyObject} from '@project-lib/core/i-any-object';
-import {Tenant} from '../../../on-boarding/models';
+import {Tenant} from '../../on-boarding/models';
```
Comment (yeshamavani) [inline projects/saas-ui/src/app/main/commands/add-tenant.command.ts:4]: why are we referring on-boarding module in main module 
move the common models and other code to shared

File: projects/saas-ui/src/app/main/components/lead-list/lead-list.component.ts [hunk 4]
```diff
@@ -4,8 +4,8 @@ import {RouteComponentBaseDirective} from '@project-lib/core/route-component-bas
 import {ColDef} from 'ag-grid-community';
 import {Location} from '@angular/common';
 import {takeUntil} from 'rxjs';
-import {OnBoardingService} from '../../on-boarding/on-boarding-service';
-import {Lead} from '../../on-boarding/models';
+import {OnBoardingService} from '../../../on-boarding/on-boarding-service';
+import {Lead} from '../../../on-boarding/models';
```
Comment (yeshamavani) [inline projects/saas-ui/src/app/main/components/lead-list/lead-list.component.ts:8]: same here also 
move common code to shared

File: projects/saas-ui/src/app/main/components/manage-plans/manage-plans.component.ts [hunk 5]
```diff
@@ -1,15 +1,15 @@
 import {Component, OnInit} from '@angular/core';
 import {ActivatedRoute, Router} from '@angular/router';
 import {ColDef, GridApi} from 'ag-grid-community';
-import {OnBoardingService} from '../../on-boarding/on-boarding-service';
+import {OnBoardingService} from '../../../on-boarding/on-boarding-service';
```
Comment (yeshamavani) [inline projects/saas-ui/src/app/main/components/manage-plans/manage-plans.component.ts:4]: not pointing everywhere 
please rectify this at all places

File: projects/saas-ui/src/app/main/commands/add-tenant.command.ts (hunk 3 above)
Comment (Deepika516) [reply projects/saas-ui/src/app/main/commands/add-tenant.command.ts:4]: Done

File: projects/saas-ui/src/app/main/commands/edit-tenant.command.ts [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import {ApiService, IAdapter, PatchAPICommand} from '@project-lib/core/api';
 
 import {IAnyObject} from '@project-lib/core/i-any-object';
-import {Tenant} from '../../model/tenant.model';
+import {Tenant} from '../model/tenant.model';
```
[the remark this answers is not in the document]
Comment (Deepika516) [reply projects/saas-ui/src/app/main/commands/edit-tenant.command.ts:4]: Done

File: projects/saas-ui/src/app/main/components/lead-list/lead-list.component.ts (hunk 4 above)
Comment (Deepika516) [reply projects/saas-ui/src/app/main/components/lead-list/lead-list.component.ts:8]: Done

File: projects/saas-ui/src/app/main/components/manage-plans/manage-plans.component.ts (hunk 5 above)
Comment (Deepika516) [reply projects/saas-ui/src/app/main/components/manage-plans/manage-plans.component.ts]: done the changes everywhere

**Review by yeshamavani — APPROVED** [review]

**Pull request merged**